baseline.yaml

```
@@ -649,6 +649,29 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+      - id: OSPS-DO-19   
+    maturity_level: 3
+    category: Documentation
+    criteria: |
+       The project MUST have performed an external security
+       review or audit within the last 4 years or after major 
+       architectural changes to understand the most likely 
+       and impactful problems that could occur within the 
+       software, and make plans to address those
+       problems.
```

```suggestion
       The project MUST have performed an external security
       review or audit within the last 4 years or after major 
       architectural changes.
```

The second half of this sounds like the objective, not the criterion.